docs/ENV.md

```diff
@@ -2,6 +2,12 @@
 
 ### `MONGO_AUTH_CREDENTIALS`
 
-Credentials to connect to your MongoDB instance.
+Credentials to connect to your MongoDB instance. You might have to get the credentials from a team member.
 
 **NOTE**: While setting this value in the `Production` and `Staging` environments on Github, make sure to include the value within double quotes and escape any characters using `\` as necessary. If you don't set this correctly, your deployment will fail.
+
+This usually looks something like this:
+
+```
+MONGO_AUTH_CREDENTIALS="mongodb+srv://COLLECTION_NAME:PASSWORD@cluster0.uocfg.mongodb.net/DATABASE_NAME?retryWrites=true&w=majority"
```

Comment:
```suggestion
MONGO_AUTH_CREDENTIALS="mongodb+srv://USER_NAME:PASSWORD@COLLECTION_NAME.uocfg.mongodb.net/DATABASE_NAME?retryWrites=true&w=majority"
```

Reply:
The `COLLECTION_NAME` is not actually a collection right?
It's a db
`quizzes`, `sessions`, `questions` etc these are collections right?